src/parents.js

```diff
@@ -43,5 +80,15 @@ export function set (node, parent, { force } = {}) {
  * @returns {object | undefined} The parent node, or undefined if the node has no parent
  */
 export function get (node) {
+	const {parent} = parentMap.get(node) ?? {};
+	return parent;
+}
+
+/**
+ * Get the parent node and metadata for a node.
+ * @param {object} node
+ * @returns {object | undefined} An object containing the parent node and the property name of the child node in the parent, or undefined if the node has no parent
+ */
+export function getDetails (node) {
```

Comment:
We probably also need `clear()` and `clearAll()` methods to implement some of the ideas below (I know we have `set(node, undefined)` but that's not super intuitive

Reply:
Yeah, was just thinking about that